store/sql_post_store.go

```diff
@@ -471,6 +473,8 @@ func (s SqlPostStore) Search(teamId string, userId string, terms string, isHasht
 				    ORDER BY CreateAt DESC
 				LIMIT 100`, searchType)
 
+			terms = strings.TrimRight(terms, specialSearchCharCutset)
```

Comment:
Postgres search and MySQL search don't return the same results as is. I think there is a much larger ticket here to go either go make them match by writing custom code or only support one fully.

Want to chat about this?